modules/fileout_netcdf/FONcArray.cc

```diff
@@ -668,6 +668,32 @@ void FONcArray::write_nc_variable(int ncid, nc_type var_type) {
     else
         d_a->intern_data(*get_eval(), *get_dds());
 
+    bool d_io_flag = d_a->get_dio_flag();
```

Comment:
For this it is better to say BESDEBUG("fonc", "d_io_flag is " << d_a->get_dio_flag() << endl); Since BESDEBUG(x); reduces to an empty statement when building for production use. When configure is used with --enable-developer.

Reply:
Yes, will do.